packages/shared/src/modules/users/libs/types/user-auth-response-dto.type.ts

```diff
@@ -1,4 +1,5 @@
 type UserAuthResponseDto = {
+	avatarUrl?: string;
```

Comment:
```suggestion
	avatarUrl: string | null;
```
it can not be optional